configuring with `--disable-png` causes builds to fail

## Upfront Information

* Fvwm3 version (run: `fvwm3 --version`)

Master

* Linux distribution or BSD name/version

Gentoo linux

* Platform (run: `uname -sp`)

N/A

## Expected Behaviour

Software with 'optional' feature disabled should build successfully.

## Actual Behaviour

Building with `./autogen.sh && ./configure --disable-png && make -j` causes unexpected build failures.

```
git clone https://github.com/fvwmorg/fvwm3.git
cd fvwm3
./autogen.sh && ./configure --disable-png && make -j
```

Error:

```
ewmh.c:288:30: error: invalid use of undefined type 'struct tm'
```
Full build log:

[build-failure-git-master.txt](https://github.com/fvwmorg/fvwm3/files/8838052/build-failure-git-master.txt)

--- a/fvwm/ewmh.c
+++ b/fvwm/ewmh.c
@@ -55,6 +55,7 @@
 #include <stdio.h>
 #include <stdarg.h>
 #include <sys/times.h>
+#include <time.h>
 #include "libs/fvwm_x11.h"
 #include "libs/fvwmlib.h"
 #include "fvwm.h"

